pkg/build/pipelines/python/build.yaml

```diff
@@ -0,0 +1,15 @@
+name: Build a Python wheel
```

Comment:
What's the difference between this and build-wheel above?

Reply:
build-wheel uses python3 specifically to build a .whl file. This `build` seems to basically just copy .py files into the right place, which is as close as Python gets to "building" things. 🤷